libs/test-common/src/main/java/org/codice/ddf/test/common/options/DistributionOptions.java

```diff
@@ -57,7 +58,9 @@ public static Option kernelDistributionOption() {
                   "bin\\start.bat",
                   "bin\\client.bat",
                   "bin\\shell.bat",
-                  "bin\\stop.bat")
+                  "bin\\stop.bat",
+                  "solr\\bin\\solr",
```

Comment:
❓ Are you sure this is still required? I would double check with the DistributionOptions class as they seem different. Same for non-windows